app/models/food.rb

```diff
@@ -0,0 +1,8 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :inventoryFoods, dependent: :destroy
+  has_many :recipeFoods, dependent: :destroy
+  validates :name, presence: true
+  validates :measurement_unit, presence: true
+  validates :price, numericality: { greater_than: 0 }
+end
```

Comment:
- [ ] Validation is not working as expected, please add the validation message when a user adds the empty data in the input fields.

Reply:
resolved 👍